examples/ibm-catalog/deployable-architectures/full-stack/README.md

```diff
@@ -1,73 +1,71 @@
 # IBM Cloud catalog example for Power infrastructure for regulated industries
 
-This example illustrates how to use the `power-infrastructure` module in the IBM catalog. It provisions the following infrastructure:
+This example provisions the following infrastructure:
+- A VPC Infrastructure with following components
+    -  3 VPCs with 1 VSI in each VPC( 1 Management(jump/bastion) vsi, 1 inet-svs vsi configured as squid proxy server, 1 private-svs vsi configured as NFS, NTP, DNS server)
+    - Installs and configures the Squid Proxy, DNS Forwarder, NTP Forwarder and NFS on hosts, and sets the host as server for these services NTP, NFS, DNS using ansible roles.
 
 - A PowerVS workspace instance with the following network topology:
-    - Creates two private networks: a management network and a backup network
-    - Creates one or two IBM Cloud connections with an option to reuse the connections
-    - Attaches the IBM Cloud connections to a transit gateway
-    - Attaches the private networks to the IBM Cloud connections
-    - Creates an SSH key
-    - Installs and configures the Squid Proxy, DNS Forwarder, NTP Forwarder and NFS on specified host, and sets the host as server for these services
+    - Creates two private networks: a management network and a backup network.
+    - Creates one or two IBM Cloud connections with an option to reuse the connections.
+    - Attaches the IBM Cloud connections to a transit gateway.
+    - Attaches the private networks to the IBM Cloud connections.
+    - Creates a SSH key.
 
-## Before you begin
+## Reference architectures
```

Comment:
I think if you want to link to the diagram only, we should rename the section to **Architecture diagram**. To leave the heading, link to the `.md` file.